packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -56,7 +57,8 @@ void main() {
     controller.text = 'test';
     await tester.idle();
     expect(tester.testTextInput.editingState['text'], equals('test'));
-    expect(tester.testTextInput.setClientArgs['inputAction'], equals(serializedActionName));
+    expect(tester.testTextInput.setClientArgs['inputAction'],
```

Comment:
nit: This file is hard to review because it contains so many unrelated format changes.